### PR TITLE
Fixes Notice: Undefined offset: 0 in BiblioStyleBase::generateBiblioMd5()

### PR DESCRIPTION
**Pull request by wundo**
#87

### CODE REVIEW

File: plugins/biblio_style/abstract.inc [hunk 1]
```diff
@@ -126,7 +126,7 @@ abstract class BiblioStyleBase implements BiblioStyleInterface {
 
         $contributors[] = array(
           $entity->biblio_contributor[LANGUAGE_NONE][0]['target_id'],
-          $entity->biblio_contributor_role[LANGUAGE_NONE][0]['target_id'],
+          empty($entity->biblio_contributor_role[LANGUAGE_NONE][0]['target_id'])? '' : $entity->biblio_contributor_role[LANGUAGE_NONE][0]['target_id'],
```
Comment (amitaibu) [inline plugins/biblio_style/abstract.inc:129]: Thanks for the PR. Why would the contributor role be empty?

**Comment by wundo** [comment]
Honestly I don't know, but this is appearing on my installation.

**Comment by wundo** [comment]
Actually, that is quite simple, as Contributor Role field is not required.

**Comment by amitaibu** [comment]
I believe the proper fix would be to make it required.

**Comment by amitaibu** [comment]
@ran81 let's make the Contributor Role field required .

**Comment by amitaibu** [comment]
Closed in favor of #93

**Pull request closed without merging**